src/tests/contract_test/wallet_test.cljs

```diff
@@ -65,3 +65,19 @@
     (fn []
       (p/let [response (contract-utils/call-rpc "wallet_getEthereumChains")]
         (assert-ethereum-chains response)))))
+
+(defn assert-address-details
+  [result]
+  (is (contains? result :address))
+  (is (contains? result :path))
+  (is (true? (:hasActivity result)))
```

Comment:
`status-go` doesn't actually check whether there is an activity in that address instead it checks whether it has any balance or not.
If the address (`test.eth`) depletes the balance, this test will fail.

I would suggest checking whether the key is present in the result or not.
OR we can check whether this value is boolean (using the `boolean?` method).

Reply:
Tks! I was not aware. Gonna do this change!